src/components/content/content.scss

```diff
@@ -1,4 +1,7 @@
 .styleguide-content {
     padding: 32px;
     position: relative;
+    min-width: calc(900px - 17px); // 17px - scrollbar width;
```

Comment:
breaks UX to develop and test mobile components.
and there is unnecessary magic. can be done easier

Reply:
@kdzuin updated diff as we agreed